serveur/Dockerfile

```
@@ -30,7 +30,7 @@ RUN --mount=type=cache,target=/opt/conda/pkgs \
 #  ----------- Instann node stuff -------------
 WORKDIR /app/serveur
 COPY serveur/package*.json ./
-RUN npm install -g npm@11.0.0
+RUN npm install -g npm@11.12.1
 RUN npm install
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)